File: content/how-to/account/examples/accountInfo.py
import alpaca_trade_api as tradeapi

api = tradeapi.REST()

# Get our account information.
account = api.get_account()

# Check if our account is restricted from trading.
can_trade = not account.trading_blocked

# Check how much money we can use to open new positions.
buying_power = account.buying_power

File: content/how-to/account/examples/accountInfo.js
const Alpaca = require('@alpacahq/alpaca-trade-api')
const alpaca = new Alpaca()

// Get our account information.
alpaca.getAccount()
    .then((account) => {
        // Check if our account is restricted from trading.
        if (account.trading_blocked) {
            console.log('Account is currently restricted from trading.')
        }

        // Check how much money we can use to open new positions.
        console.log(`$${account.buying_power} is available as buying power.`)
    })